Tests/StencilSwiftKitTests/TestsHelper.swift

```diff
@@ -42,7 +45,14 @@ func diff(_ result: String, _ expected: String) -> String? {
     }
     let lhsNum = addLineNumbers(slice(lhsLines, 4)).joined(separator: "\n")
     let rhsNum = addLineNumbers(slice(rhsLines, 4)).joined(separator: "\n")
-    return "\(msgColor)Mismatch at line \(badLineIdx)\(reset)\n>>>>>> result\n\(lhsNum)\n======\n\(rhsNum)\n<<<<<< expected"
+    return [
+      "\(msgColor)Mismatch at line \(badLineIdx)\(reset)\n",
```

Comment:
Extra `\n` (which should be added by the `join`: is it intentional (so that it ends up with two \n for this line and thus a blank line in the result) or not?